.github/workflows/test.yml

```
@@ -1,7 +1,21 @@
 name: MineAdmin Unit and Ci Test
 on:
   push:
+    paths:
+        - 'app/**'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)